framework/Csv_loader.py

```diff
@@ -207,39 +222,52 @@ def __csvLoaderForPoint(self,fileIn,options):
         for key in outParam:
           if key in self.allFieldNames:
             outDict[key] = np.atleast_1d(np.array(data[lastRow,self.allFieldNames.index(key)]))
-          else: self.raiseAnError(IOError,"the parameter " + str(key) + " has not been found")
+          else:
+            self.raiseAnError(IOError,"the parameter " + str(key) + " has not been found")
     elif outputRow != None:
-      if outputRow > data[:,0].size-1  and outputRow != -1: self.raiseAnError(IOError,'outputRow is greater than number of actual rows in file '+ str(fileIn) + '!')
+      if outputRow > data[:
+        ,0].size-1  and outputRow != -1: self.raiseAnError(IOError,'outputRow is greater than number of actual rows in file '+ str(fileIn) + '!')
```

Comment:
I think all these could be fixed if the script checked if the next non-whitespace character was not a comma.